manual/manual/cmds/intf-c.etex

```diff
@@ -1553,31 +1553,37 @@ to the extension of the resulting file (given with "-o").
 The native-code compiler "ocamlopt" also supports the "-output-obj"
 option, causing it to output a C object file or a shared library
 containing the native code for all OCaml modules on the command-line,
-as well as the OCaml startup code. Initialization is performed by
-calling "caml_startup" as in the case of the bytecode compiler.
+as well as the OCaml startup code and the OCaml runtime system.
```

Comment:
I don't think it formally contains the runtime system. You still have to link against `libasmrun.a`.